docs/server/source/cloud-deployment-templates/node-on-kubernetes.rst

```diff
@@ -128,21 +133,21 @@ read-write by a single Kubernetes node.
 by AzureDisk.)
 ``storage: 20Gi`` means the volume has a size of 20
 `gibibytes <https://en.wikipedia.org/wiki/Gibibyte>`_.
-(You can change that if you like.)
 
-Create ``mongoclaim`` in your Kubernetes cluster:
+You may want to update the ``spec.resources.requests.storage`` field in both
+the files to specify a different disk size.
 
-.. code:: bash
+Create the required PersistentVolumeClaim using:
 
-   $ kubectl apply -f mongoclaim.yml
+.. code:: bash
 
-You can check its status using:
+   $ kubectl apply -f mongo-data-db-pvc.yaml
+   $ kubectl apply -f mongo-data-configdb-pvc.yaml
 
-.. code:: bash
 
-   $ kubectl get pvc
+You can check its status using: ``kubectl get pvc -w``
 
-Initially, the status of ``mongoclaim`` might be "Pending"
+Initially, the status of persistent volume claims might be "Pending"
```

Comment:
Please remove or update the block of text below this (i.e. showing `$ kubectl describe pvc` and its output): it is out-of-date.